core/livepeernode.go

```diff
@@ -259,6 +259,12 @@ func (n *LivepeerNode) transcodeAndBroadcastSeg(seg *stream.HLSSegment, sig []by
 		return // TODO return error?
 	}
 
+	// Ensure length matches expectations. 4 second + 25% wiggle factor, 60fps
+	err := ffmpeg.CheckMediaLen(fname, 4*1.25*1000, 60*4*1.25)
+	if err != nil {
+		glog.Errorf("Media length check failed: %v", err)
+		return // XXX should we give up on the job entirely here?
```

Comment:
Yup, the concern was more about the transcoder being able to skip claim ranges for invalid segments; we don't want to do more work than necessary if we know the job is invalid from the outset. But since transcoders can indeed skip claims, this is a non-issue. Will remove the comment.

Reply:
Fixed, comment removed.